core/src/main/java/com/sequenceiq/cloudbreak/core/flow/context/StackScalingContext.java

```diff
@@ -6,6 +6,7 @@
 import com.sequenceiq.cloudbreak.domain.CloudPlatform;
 import com.sequenceiq.cloudbreak.domain.Resource;
 import com.sequenceiq.cloudbreak.domain.ScalingType;
+import com.sequenceiq.cloudbreak.service.stack.event.RemoveInstanceRequest;
 import com.sequenceiq.cloudbreak.service.stack.event.UpdateInstancesRequest;
 
 public class StackScalingContext extends DefaultFlowContext implements FlowContext {
```

Comment:
I think a new Context class should be created for the remove-instance flow. InstanceId is not used during the scaling flows, and the other fields are not really used during the remove-instance flow.